po/it.po

```diff
@@ -1886,34 +1888,35 @@ msgid ""
 "unblocks blocked episodes (to block all then unblock some)."
 msgstr ""
 "Facendo clic sul blocco spunte e lasciandolo vuoto disabiliterà l'auto-"
-"scaricamento per tutti gli episodi in questo canale.  I modelli "
-"corrispondono al testo parziale nel titolo dell'episodio, e uno schema vuoto "
-"corrisponde a qualunque titolo. Il modello eccetto sblocca gli episodi "
-"bloccati (per bloccare tutti e poi sbloccarne alcuni)."
+"scaricamento per tutti gli episodi in questo canale. \n"
+"I modelli corrispondono al testo parziale nel titolo dell'episodio, e uno "
+"schema vuoto corrisponde a qualunque titolo. \n"
+"Il modello eccetto sblocca gli episodi bloccati (per bloccare tutti e poi "
+"sbloccarne alcuni)."
```

Comment:
Does the last sentence indicate it blocks or unblocks filtered episodes, or that it locks or unlocks them like the archive menu item?

Reply:
Please take care that there is a pending issues about italian language update

https://github.com/gpodder/gpodder/pull/1537

Reply:
What?

Reply:
We cannot discuss about an old version of translation until author not include latest pending integration of italian tarnslation.

Reply:
The translation can not be merged until these review questions have been answered.

Reply:
@auouymous 

I forgot that you are the author (I beleive .... ;-)

Itunblock the episodes basing on exception model.

I will do a small change in

"Il modello eccezione sblocca gli episodi bloccati (per bloccarli tutti e poi "
"sbloccarne soilo alcuni)."

Thanks.

Note:; if you can please avoid in .poo file in teh string the automatic wordwrap at 80 chars:
It cannot help on long strings.
What's teh comamnd to sue to format the string in .po file?

Reply:
> I forgot that you are the author

You should answer any review questions, even if the user isn't associated with the project. Members can close irrelevant questions, but valid issues must be resolved, no matter who asks them.

> please avoid in .poo file in teh string the automatic wordwrap at 80 chars:
> What's teh comamnd to sue to format the string in .po file?

@elelay Do you know the answer to that?

Reply:
commands are run from the makefile. `msgmerge` has a `--no-word-wrap` option but I don't see why we should use it. @bovirus what's your problem again?

Reply:
@elelay

The problem is about long string.
in your .po files seems wordwrap automatically at 80 chars.
Then long string are wordwrapped and when you translate it if you sue a simple text editor (and not POEdit) is mroe complex to revise and edit string.
For transltor can help to have long string without wordwrao at 80 chars.

Reply:
@elelay elelay

Example

** Now

msgid ""
"Clicking the block checkbox and leaving it empty will disable auto-download "
"for all episodes in this channel.  The patterns match partial text in "
"episode title, and an empty pattern matches any title.  The except pattern "
"unblocks blocked episodes (to block all then unblock some)."
msgstr ""
"Facendo clic sul blocco spunte e lasciandolo vuoto disabiliterà l'auto-"
"scaricamento per tutti gli episodi in questo canale. \n"
"I modelli corrispondono al testo parziale nel titolo dell'episodio, e uno "
"schema vuoto corrisponde a qualunque titolo. \n"
"Il modello eccetto sblocca gli episodi bloccati (per bloccare tutti e poi "
"sbloccarne alcuni)."

** My idea (without word-wrap)

msgid "Clicking the block checkbox and leaving it empty will disable auto-download for all episodes in this channel. \nThe patterns match partial text in episode title, and an empty pattern matches any title.\nThe except pattern unblocks blocked episodes (to block all then unblock some).
msgstr "Facendo clic sul blocco spunte e lasciandolo vuoto disabiliterà l'auto-scaricamento per tutti gli episodi in questo canale.\nI modelli corrispondono al testo parziale nel titolo dell'episodio, e uno schema vuoto corrisponde a qualunque titolo. \nIl modello eccetto sblocca gli episodi bloccati (per bloccare tutti e poi sbloccarne alcuni)."

I hope that is now more clear.